website/app/styles/doc-components/image-caption.scss

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// IMAGE-CAPTION
+
+@use "../typography/mixins" as *;
+
+.doc-image-caption {
+    @include doc-font-style-body-small();
+    margin-top: 4px;
+    color: var(--doc-color-gray-300);
+    font-style: italic;
+    letter-spacing: 0.0125em;
```

Comment:
We don't change the letter spacing or kerning in the fonts we use (it's a Figma thing)
```suggestion
```

Reply:
ok